plugin/buycourses/src/course_panel.php

```diff
@@ -39,23 +44,17 @@
     }
 }
 
-$toolbar = Display::toolbarButton(
-    $plugin->get_lang('CourseListOnSale'),
-    'course_catalog.php',
-    'search-plus',
-    'primary',
-    ['title' => $plugin->get_lang('CourseListOnSale')]
-);
+$interbreadcrumb[] = ['url' => '../index.php', 'name' => $plugin->get_lang('UserPanel')];
```

Comment:
This breadcrumb (over the whole plugin) is a mess. We need to make sure the first part is always /plugin/buycourses/index.php for the admin, and /plugin/buycourses/src/course_catalog.php for all other users.